src/util/apiUtils.js

```diff
@@ -589,7 +593,7 @@ export const injectResponseCookies = request => ([response, _, jar]) => {
 			path: cookie.path,
 			isHttpOnly: cookie.httpOnly,
 			isSameSite: false,
-			isSecure: process.env.NODE_ENV === 'production',
+			isSecure: config.isProd,
```

Comment:
@mmcgahan would you happen to know why `request.server.settings` is not available here?